testsuite/oslc-warn-commainit/test.osl

```diff
@@ -0,0 +1,5 @@
+shader test ( output color Cout = 0 )
+{
+    vector x = (1, 2, 3);
```

Comment:
That's already a syntax error.

I'm thinking about allowing aggregate assignment in that way, but it's a much bigger grammatical and implementation change than simply warning about the one thing people seem to commonly mistake.